spec/unit/forge/connection/connection_failure_spec.rb

```diff
@@ -21,6 +22,12 @@
     }.to raise_error(Faraday::ConnectionFailed, /unable to connect to.*\/connectfail.*name or service not known/i)
   end
 
+  it "logs for timeout errors" do
+    expect {
+      subject.get('/timeout')
+    }.to raise_error(Faraday::ConnectionFailed, /unable to connect to.*\/timeout.*request timed out/i)
```

Comment:
I assume you meant to test `Faraday::TimeoutError` here? since `ConnectionFailed` was the previous behavior.

Reply:
No, we still raise a `ConnectionFailed` error when we re-raise the `TimoutError` from Faraday. I decided to contain the API breakage here and let consumers of the Forge gem continue as they always did, rather than hitting this "regression" from the curl API change. The fact a timeout occurred is reflected in the error message.

But I'm open to the idea of changing the type raised if people think that would be better. Might want some input from the maintainers of this gem before doing that though.

Reply:
oh I see, I hadn't expanded that `rescue` to see what actually happened below. This seems good to me then. 👍